matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/RoomPinnedEventsContent.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.matrix.androidsdk.rest.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Content of a m.room.pinned_events type event
+ */
+public class RoomPinnedEventsContent implements Serializable {
+
+    public List<String> pinned;
```

Comment:
Please add a comment like below:
`// List of eventIds of pinned events` or something similar